config/crd/bases/infrastructure.cluster.x-k8s.io_haproxyloadbalancers.yaml

```diff
@@ -129,7 +129,7 @@ spec:
                             ipAddrs:
                               description: IPAddrs is a list of one or more IPv4 and/or
                                 IPv6 addresses to assign to this device. Required
-                                when DHCP4 and DHCP6 are both false.
+                                when DHCP4 and DHCP6 are both false. Written in CIDR notation.
```

Comment:
comments needs to be updated direcly on the types. then running `make generate` will update the CRDs automatically

Reply:
Would this only be in api/v1alpha3/types.go ?

Reply:
Yes